fix(orc8r): Revert additonal labels from k8s services to avoid incompatibilities with deployed objects

joary opened the pull request:
An existing service has imutable selectors, the additonal labels included in PR10085 creates an incompatibility with existing services.

Signed-off-by: Joary Paulo Wanzeler Fortuna <joarypl@fb.com>

## Summary

Disable the labels added to k8s services objects from PR10085

## Test Plan


## Additional Information

- [ ] This change is backwards-breaking

<!--
    If this is a backwards-breaking change, document the upgrade instructions.
    All upgrade instructions for backwards-breaking changes will be aggregated
    in the next release's changelog so this is very important to fill out.
-->

uri200 commented:
can you take a look if  there is something else from this PR https://github.com/magma/magma/pull/10792 that needs to be reverted please and add it here?

joary committed:
fix(orc8r): version bump orc8r helm chart

Signed-off-by: Joary Paulo Wanzeler Fortuna <joarypl@fb.com>

joary committed:
fix(orc8r): Revert additonal labels from k8s services to avoid incompatibilities with deployed objects

An existing service has imutable selectors, the additonal labels included in PR10085 creates an incompatibility with existing services.

Signed-off-by: Joary Paulo Wanzeler Fortuna <joarypl@fb.com>
